packages/nightingale-manager/README.md

```diff
@@ -0,0 +1,52 @@
+# nightingale-manager
+
+[![Published on NPM](https://img.shields.io/npm/v/nightingale-manager.svg)](https://www.npmjs.com/package/nightingale-manager)
+
+It is a container for all the tracks and other inner components (e.g. protvista-navigation).
+
+This component is in charge of capturing the events that its protvista descendent trigger, and adjust element attributes accordingly.
+
+A protvista descendent is any element in its subtree which tag name starts with 'protvista'.
+
+The children should fire events of the type change, and the detail object should contain the attributes `type` and `value`.
+
+Additionally, there are a set of pre-defined attributes which can be used with this component and will be propagated to descendents: `displaystart`, `displayend`, `highlight` and `length`. This is useful if you want to control things like zoom level from outside, for instance directly from your application.
+
+See it running [Here](https://ebi-webcomponents.github.io/nightingale/#/manager).
+
+## Usage
+
+```html
+<nightingale-manager attributes="attr1 attr2"></nightingale-manager>
+```
+
+Here is an example of how the child components should fire the events:
+
+```javascript
+this.dispatchEvent(
+  new CustomEvent("change", {
+    detail: {
+      value: "New value",
+      type: "attr1",
+    },
+  })
+);
+```
+
+## API Reference
```

Comment:
I'm been a bit picky here, but maybe we should take this whole process of migrating into typescript to make sure the API documentation of each component is complete.